<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1402</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="HeaterLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Heater</string>
    </property>
    <widget class="QPushButton" name="HeaterSwitch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>   |    O</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="HeaterScrollBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="HeaterStatus">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LightBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Light</string>
    </property>
    <widget class="QPushButton" name="LightButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>  |    O</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="LightProgressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="LowBrightness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Low Light</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="MediumBrightness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Medium Light</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="HighBrightness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>High Light</string>
     </property>
    </widget>
    <widget class="QLabel" name="LightError">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Fan</string>
    </property>
    <widget class="QPushButton" name="FanSwitch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>   |    O</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Low</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Medium</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>High</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Exhaust</string>
    </property>
    <widget class="QPushButton" name="ExhaustSwitch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>   |    O</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ToggleSwitch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Open/Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>20</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>WaterPump</string>
    </property>
    <widget class="QPushButton" name="WaterPumpSwitch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>   |    O</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="PumpRateFlow">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>30</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Irrigation</string>
    </property>
    <widget class="QPushButton" name="IrrigationSwitch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>   |    O</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Temperature Sensor</string>
    </property>
    <widget class="QTextEdit" name="TemperatureInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperature:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Celsius</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Humidity Sensor</string>
    </property>
    <widget class="QTextEdit" name="HumidityInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Humidity Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>270</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Illumination Sensor</string>
    </property>
    <widget class="QTextEdit" name="IlluminationInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Illumination Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lux</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>280</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Soil Moisture Sensor</string>
    </property>
    <widget class="QTextEdit" name="IlluminationInput_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Soil Moisture Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1402</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
